docs/notebooks/running_test_application.ipynb

```diff
@@ -0,0 +1,354 @@
+    "# connect to the API\n",
+    "from sentinelsat import SentinelAPI, read_geojson, geojson_to_wkt\n",
+    "from datetime import date\n",
+    "user = 'username'\n",
+    "password = 'password'\n",
+    "# initialize settings\n",
+    "api = SentinelAPI(user, password, 'https://scihub.copernicus.eu/apihub/')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# search by polygon (MNI test site coordinates), time, and SciHub query keywords\n",
+    "footprint = geojson_to_wkt(read_geojson('coordinates_mni.geojson'))\n",
+    "products = api.query(footprint,\n",
+    "                     date=('20200101', '20200120'),\n",
+    "                     platformname='Sentinel-1',\n",
+    "                     producttype='SLC')\n",
+    "print('Following products will be downloaded')\n",
+    "print(api.to_dataframe(products).title.values)\n",
+    "\n",
+    "print('These {} product need {} Gb of disk space'.format(len(products), api.get_products_size(products)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Start download process (<span style=\"color:red\">Attention: might take a while and data will requries some free disk space</span>)**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# download all results from the search\n",
+    "# files will be downloaded to specified path\n",
+    "import os\n",
+    "path = \"/path/to/data/\"\n",
+    "try:\n",
+    "    os.makedirs(path)\n",
+    "except: FileExistsError\n",
+    "api.download_all(products, path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Use sar-pre-processing package to process Sentinel-1 SLC data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set paths for\n",
+    "- input_folder (location with stored Sentinel-1 SLC data)\n",
+    "- output_folder (location where processed data will be stored)\n",
+    "- gpt_loction (gpt is located in the bin folder of your SNAP installation)' "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_folder = path\n",
+    "output_folder = path\n",
+    "gpt_location = os.path.expanduser('~/snap/bin/gpt')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create config file with information about input, output and gpt loction"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import yaml\n",
+    "\n",
+    "with open('sample_config_file.yaml') as stream:\n",
+    "   data = yaml.safe_load(stream)\n",
+    "\n",
+    "data['input_folder'] = input_folder\n",
+    "data['output_folder'] = output_folder\n",
+    "data['gpt'] = gpt_location\n",
```

Comment:
I'd love if you'd state how to change more parameters, namely year and bounding box.

Reply:
included